libcontainer/configs/cgroup_unix.go

```diff
@@ -33,6 +37,8 @@ type Cgroup struct {
 	*Resources
 }
 
+const MemoryUnlimited = math.MaxUint64
```

Comment:
Also this value may well be totally incorrect on 32 bit machines, not sure what happens there.